appserv-jstl: Use xalan (org.apache.*) instead of Sun JDK internal copy
(com.sun.org.apache.*.internal.*)

--- origsrc/glassfish/appserv-jstl/src/org/apache/taglibs/standard/tag/common/xml/JSTLXPathFactory.java	2008-04-17 13:59:44.000000000 -0500
+++ src/glassfish/appserv-jstl/src/org/apache/taglibs/standard/tag/common/xml/JSTLXPathFactory.java	2010-06-03 05:19:25.125354900 -0500
@@ -54,7 +54,7 @@
 
 package org.apache.taglibs.standard.tag.common.xml;
 
-import com.sun.org.apache.xpath.internal.jaxp.XPathFactoryImpl;
+import org.apache.xpath.jaxp.XPathFactoryImpl;
 
 /**
  * This factory class is added to provide access to our own implementation
--- origsrc/glassfish/appserv-jstl/src/org/apache/taglibs/standard/tag/common/xml/JSTLXPathImpl.java	2008-04-17 13:59:44.000000000 -0500
+++ src/glassfish/appserv-jstl/src/org/apache/taglibs/standard/tag/common/xml/JSTLXPathImpl.java	2010-06-03 05:19:25.156356700 -0500
@@ -62,11 +62,11 @@ import javax.xml.xpath.XPathFunctionReso
 import javax.xml.xpath.XPathVariableResolver;
 import javax.xml.xpath.XPathExpression;
 
-import com.sun.org.apache.xml.internal.dtm.DTM;
-import com.sun.org.apache.xpath.internal.*;
-import com.sun.org.apache.xpath.internal.objects.XObject;
-import com.sun.org.apache.xpath.internal.res.XPATHErrorResources;
-import com.sun.org.apache.xalan.internal.res.XSLMessages;
+import org.apache.xml.dtm.DTM;
+import org.apache.xpath.*;
+import org.apache.xpath.objects.XObject;
+import org.apache.xpath.res.XPATHErrorResources;
+import org.apache.xalan.res.XSLMessages;
 
 import org.w3c.dom.Node;
 import org.w3c.dom.DOMImplementation;
@@ -100,7 +100,7 @@ public class JSTLXPathImpl implements ja
     private XPathVariableResolver origVariableResolver;
     private XPathFunctionResolver origFunctionResolver;
     private NamespaceContext namespaceContext=null;
-    private com.sun.org.apache.xpath.internal.jaxp.JAXPPrefixResolver prefixResolver;
+    private org.apache.xpath.jaxp.JAXPPrefixResolver prefixResolver;
     // By default Extension Functions are allowed in XPath Expressions. If 
     // Secure Processing Feature is set on XPathFactory then the invocation of
     // extensions function need to throw XPathFunctionException
@@ -179,7 +179,7 @@ public class JSTLXPathImpl implements ja
             throw new NullPointerException( fmsg ); 
         }
         this.namespaceContext = nsContext;
-        this.prefixResolver = new com.sun.org.apache.xpath.internal.jaxp.JAXPPrefixResolver ( nsContext );
+        this.prefixResolver = new org.apache.xpath.jaxp.JAXPPrefixResolver ( nsContext );
     }
 
     /**
@@ -230,21 +230,21 @@ public class JSTLXPathImpl implements ja
     
     private XObject eval(String expression, Object contextItem)
         throws javax.xml.transform.TransformerException {
-        com.sun.org.apache.xpath.internal.XPath xpath = new com.sun.org.apache.xpath.internal.XPath( expression,
-            null, prefixResolver, com.sun.org.apache.xpath.internal.XPath.SELECT ); 
-        com.sun.org.apache.xpath.internal.XPathContext xpathSupport = null;
+        org.apache.xpath.XPath xpath = new org.apache.xpath.XPath( expression,
+            null, prefixResolver, org.apache.xpath.XPath.SELECT ); 
+        org.apache.xpath.XPathContext xpathSupport = null;
         if ( functionResolver != null ) {
-            com.sun.org.apache.xpath.internal.jaxp.JAXPExtensionsProvider jep = 
-                    new com.sun.org.apache.xpath.internal.jaxp.JAXPExtensionsProvider(
+            org.apache.xpath.jaxp.JAXPExtensionsProvider jep = 
+                    new org.apache.xpath.jaxp.JAXPExtensionsProvider(
                     functionResolver, featureSecureProcessing );
-            xpathSupport = new com.sun.org.apache.xpath.internal.XPathContext( jep );
+            xpathSupport = new org.apache.xpath.XPathContext( jep );
         } else { 
-            xpathSupport = new com.sun.org.apache.xpath.internal.XPathContext();
+            xpathSupport = new org.apache.xpath.XPathContext();
         }
 
         XObject xobj = null;
         
-        xpathSupport.setVarStack(new com.sun.org.apache.xpath.internal.jaxp.JAXPVariableStack(variableResolver));
+        xpathSupport.setVarStack(new org.apache.xpath.jaxp.JAXPVariableStack(variableResolver));
         
         // If item is null, then we will create a a Dummy contextNode
         if ( contextItem instanceof Node ) {
@@ -372,7 +372,7 @@ public class JSTLXPathImpl implements ja
         }
         // JSTLXPathConstants.OBJECT
         if ( returnType.equals( JSTLXPathConstants.OBJECT ) ) {
-            if (resultObject instanceof com.sun.org.apache.xpath.internal.objects.XNodeSet)
+            if (resultObject instanceof org.apache.xpath.objects.XNodeSet)
                 return resultObject.nodelist();
             else 
                 return resultObject.object();
